api/chains/models.go

```diff
@@ -95,6 +95,23 @@ type InflationResponse struct {
 	Inflation string `json:"inflation"`
 }
 
+type StakingParamsResponse struct {
+	Params struct {
+		UnbondingTime     int64  `json:"unbonding_time"`
+		MaxValidators     int64  `json:"max_validators"`
+		MaxEntries        int64  `json:"max_entries"`
+		HistoricalEntries int64  `json:"historical_entries"`
+		BondDenom         string `json:"bond_denom"`
+	}
+}
+
+type StakingPoolResponse struct {
+	Pool struct {
+		NotBondedTokens string `json:"not_bonded_tokens"`
+		BondedTokens    string `json:"bonded_tokens"`
+	}
```

Comment:
is it correct that it's missing the json tag here?

```suggestion
	} `json:"pool"`
```